### ISSUE TITLE
Clean up the recent log messages table even/odd styles

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#876]
This is a follow-up from https://github.com/backdrop/backdrop-issues/issues/739. The same icons are used both for messages and for recent log messages. We adjusted the background colors to match the messages, but there is now some color clashing between the "Seven red" and the "Error red":

![recent log messages l backdropdev com](https://cloud.githubusercontent.com/assets/5607236/6627669/2f4eb6a0-c8d6-11e4-8452-f24c81da8472.png)

Other even/odd rows have similar styling problems. We may need design direction on this issue from @dariusgarza.

**Comment by quicksketch** [comment]
After merging in the table styles, this looks quite a bit better, but it still needs some consideration. It makes no sense why some cells are green or blue while the first column is either lacking color entirely or is yellow:

![error-log](https://cloud.githubusercontent.com/assets/100206/7218184/d4cf0a80-e613-11e4-9ed1-f552b5613580.png)

For those curious, there are actually *seven* levels of errors, which is probably where some of these oddball colors originate. From watchdog_severity_levels():

```
    WATCHDOG_EMERGENCY   => t('emergency'),
    WATCHDOG_ALERT   => t('alert'),
    WATCHDOG_CRITICAL    => t('critical'),
    WATCHDOG_ERROR     => t('error'),
    WATCHDOG_WARNING => t('warning'),
    WATCHDOG_NOTICE  => t('notice'),
    WATCHDOG_INFO    => t('info'),
    WATCHDOG_DEBUG   => t('debug'),
```

Should try to account for these different levels? Or should we just group them into buckets?

- Debug, Info, Notice: Blue background. "Info" icon.
- Warning: Yellow background. "Warning" icon.
- Error, Critical, Alert, Emergency: Red background. "Error" icon.

Note that for the most part, Backdrop does not use info, critical, alert, or emergency levels. However we have them to line up with standard syslog levels in BSD: http://www.ietf.org/rfc/rfc3164.txt.

**Comment by klonos** [comment]
> ... Or should we just group them into buckets?

Buckets please! Will be easier to implement too. :wink: 

We should have two shades of each color (in order to account for ```.even```/```.odd```). Not sure if we need different color for ```.active```?

**Comment by docwilmot** [comment]
Three buckets, please. That kaleidoscope up there is not gonna work out.

**Comment by Drupaldise** [comment]
Why do we not remove the background color from the first col or change the icon?
I am not sure if the bucket solution makes the log better readable.

**Comment by wesruv** [comment]
Funny story, more date columns would be in a color other than gray except the `tr.odd` class has a higher specificity than the status bg colors. *facepalm*

Anyway I took a quick stab at doing something a little nicer that won't involve too much time/work. My gut is that we shouldn't spend too much time on this page at this point. Not say it's not important, I just think there are more impactful pages to the user experience that we could give some UI love.
![recent log messages l backdropdev com](https://cloud.githubusercontent.com/assets/5607236/7266944/9d17d152-e880-11e4-9212-34ea63f4da1b.png)

**Comment by quicksketch** [comment]
I think this looks great @wesruv. @dariusgarza could you give this a thumbs-up if needed?

I also just realized that we have this exact same icons problem on the Status Report page. Perhaps the same solution would work there as well?

**Comment by wesruv** [comment]
I'll take a look!

**Comment by wesruv** [comment]
@quicksketch Here's a PR with the circle enclosure changes for log messages and status pages:
https://github.com/backdrop/backdrop/pull/915

**Comment by quicksketch** [comment]
This looks pretty good. Do you think we could add in the icon for "info", "notice", and "debug" as well? It looks incorrect when there are icons for some rows but not others. This isn't a new problem but it would be great to get addressed at the same time.

**Comment by wesruv** [comment]
@quicksketch, agreed, are there icons for these?

Notice seems like a triangle would work, maybe ( i ) for the other two?

**Comment by quicksketch** [comment]
> Notice seems like a triangle would work, maybe ( i ) for the other two?

That sounds great. Considering PHP "Notices" really should be visually displayed as warnings at the very least. WATCHDOG_WARNING and WATCHDOG_NOTICE are next to each other in the spectrum

**Comment by wesruv** [comment]
@quicksketch updated PR: https://github.com/backdrop/backdrop/pull/915

**Comment by wesruv** [comment]
@quicksketch PR has been updated.

**Comment by quicksketch** [comment]
Thanks, the new PR looks great! Merged into 1.1.x and 1.x, so this will be in 1.1.2. Great job!

**Comment by klonos** [comment]
Really great indeed. Thanx!!
